android/src/paper/java/com/facebook/react/viewmanagers/RNSSearchBarManagerDelegate.java

```diff
@@ -74,7 +74,7 @@ public void setProperty(T view, String propName, @Nullable Object value) {
   }
 
   @Override
-  public void receiveCommand(T view, String commandName, ReadableArray args) {
+  public void receiveCommand(T view, String commandName, @Nullable ReadableArray args) {
```

Comment:
This was most likely caused by update of codegen version alongside react native or react-native-cli (Idk where codegen lies right now).